src/ceph-volume/ceph_volume/util/prepare.py

```diff
@@ -76,7 +76,7 @@ def create_id(fsid, json_secrets, osd_id=None):
     return ' '.join(stdout).strip()
 
 
-def check_id(osd_id):
+def check_id_exists(osd_id):
```

Comment:
could you make this:

    def osd_id_exists(osd_id):

Otherwise it reads odd when used in the logic checking the boolean value

Reply:
Good point.  I have pushed the function-name change.

Also I re-arranged the IF test, placing the ( osd_id is not None ) before ( osd_id_exists(osd_id) ).  This avoids an unnecessary call to "ceph osd tree", when osd_id is NONE

Do you need me to send TEST function name changes to test_prepare.py in this PR, or a separate PR?